app/controllers/carts_controller.rb

```diff
@@ -16,6 +18,13 @@ def index
     @items = @cart.items
   end
 
+  def update
+    @cart.update_item(cart_params)
```

Comment:
@tmikeschu I'm a little confused about what's going on with the `cart_params` so I have a couple questions.

1. Why do we need that `before_action`?
2. Why do we need to `permit` since the params aren't nested at all?

I checked out a branch and replaced `cart_params` with just `params` and all tests still passed.

![screen shot 2017-01-05 at 7 08 37 pm](https://cloud.githubusercontent.com/assets/16868275/21705049/a0ed856a-d37a-11e6-9e18-25f155547b6c.png)